newdle/config.py

```diff
@@ -0,0 +1,12 @@
+from flask import Blueprint, current_app
+
+
+config = Blueprint('config', __name__, url_prefix='/config')
+
+
+@config.route('/footer_links/')
+def footer_links():
+    if 'FOOTER_LINKS' in current_app.config:
```

Comment:
Put an empty default value (plus documentation like we have for the other config settings) in `newdle.cfg.example` - that file is always loaded before the real config, so we know for sure that any config key is defined.